.github/workflows/lint-test.yaml

```diff
@@ -0,0 +1,49 @@
+name: Lint and Test Charts
+
+# yamllint disable-line rule:truthy
+on:
+  workflow_call:
+  workflow_dispatch:
+  pull_request:
+    paths:
+    - 'charts/**'
+
+jobs:
+  lint-test:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
```

Comment:
I think fetch depth 0 is the default

Reply:
> Only a single commit is fetched by default, for the ref/SHA that triggered the workflow. Set fetch-depth: 0 to fetch all history for all branches and tags. Refer [here](https://docs.github.com/actions/using-workflows/events-that-trigger-workflows) to learn which commit $GITHUB_SHA points to for different events.

ref: https://github.com/actions/checkout/#checkout-v4

`fetch-depth: 0` is required by `ct` in order to compute the diff against the main branch and figure out changes/tag bumps etc.